common/app/views/fragments/amp/stylesheets/tones.scala.html

```diff
@@ -21,7 +21,8 @@
     color: #fff;
 }
 
-.tonal--tone-comment .tonal__standfirst {
+.tonal--tone-comment .tonal__standfirst,
```

Comment:
PS the spec says:

Allowed selectors are of the form:
`.class`
`#id`
`tag-name`
`selector, selector`
`media queries`